ui/src/utils/api.js

```diff
@@ -11,12 +11,20 @@ const transformJob = ({ jobId, title, description, hasStar, keywords, href }) =>
 }
 );
 
+const transformPagination = ({ number, totalPages }) => ({
+  page: number + 1,
+  totalPages
+});
+
 const prefix = '/api';
 
-export const fetchJobs = () => (
-  fetch(`${prefix}/jobs?size=100`)
+export const fetchJobs = (page = 1) => (
+  fetch(`${prefix}/jobs?size=100&page=${page - 1}`)
```

Comment:
page - 1? why?

Reply:
Counting from 0 on backend and counting from 1 on frontend.

Reply:
There is a way to change pagintation default, so it will start from 1 instead of 0
* https://github.com/spring-projects/spring-boot/issues/14413
* https://github.com/spring-projects/spring-boot/issues/14413#issuecomment-485069382
* https://stackoverflow.com/a/49575492/2051256
* https://jira.spring.io/browse/DATAREST-1290

Reply:
This partially works. When sending request to backend it works - page 1 is the first page. But in response there is `number` which is still 0 based. E.g. Request for `page=3` correctly returns third page, but `number` in response has value 2.

Hmm...

Reply:
Maybe it will work:

```java
@Configuration
public class PageableConfig {

    @Bean
    PageableHandlerMethodArgumentResolverCustomizer pageableResolverCustomizer() {
        return pageableResolver -> pageableResolver.setOneIndexedParameters(true);
    }
}
```

Reply:
This is exactly what I have right now. `application.properties` does not change setting at all.

Reply:
https://jira.spring.io/browse/DATACMNS-563

Reply:
I added simple `+1` in `PageResponse`. It's a bit hacky but is also future proof (from prespective of upgrading spring). 

I'm not sure why but when I removed `spring.data.web.pageable.one-indexed-parameters: true` it stopped interpreting parameters with indexing from one. So we have this setting in `application.yml` and also in `PageableConfig` just for sure.